### DISCUSSION TITLE
Restore from snapshot / expand volume

### DISCUSSION THREAD

**Discussion by runningman84** [longhorn/longhorn#8105] [General]
I have restored a volume from a backup/snapshot. In k8s it is shown as longhorn-static storage class.

I cannot expand the volume:
"unable to expand volume foo-config: persistentvolumeclaims "foo-config" is forbidden: only dynamically provisioned pvc can be resized and the storageclass that provisions the pvc must support resize"

How can I fix this?

I tried to restore it from the snapshot but I cannot choose a bigger size there...

**Comment by derekbit** [comment]
Please check https://github.com/longhorn/longhorn/issues/6446
